x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -1141,7 +1144,7 @@ Collect and make events from response in any format supported by httpjson for al
 
 The `replace_with: "pattern,value"` clause is used to replace a fixed pattern string defined in `request.url` with the given value. 
 The fixed pattern must have a `$.` prefix, for example: `$.xyz`. The `value` may be hard coded or extracted from context variables 
-like [`.last_response.*`, `.first_response.*`] etc. The `replace_with` clause can be used in combination with the `replace` clause 
+like [`last_response.*`, `first_response.*`, `parent_last_response.*`] etc. The `replace_with` clause can be used in combination with the `replace` clause 
```

Comment:
Not for this PR.

This is not a new behaviour, but do you know the reason for it? It matches the behaviour of `fetchValueFromContext` but that could be altered to look at the first word after a dot rather than the first element of a split on dot which would then mean that the syntax would be consistent with the use of these in templates.

Reply:
Yes we could definitely iterate and improve that behaviour, and it makes more sense.